irs/src/components/Login/Register.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Col, Row, Button, Form, FormGroup, Label, Input, FormText } from 'reactstrap';
+
+class Register extends React.Component {
+  state = { 
+    credentials: {
+      username: "",
+      password: "",
+      authLevel: "",
+      organization: "",
+    },
+    data: {
+      firstName: "",
+      lastName: "",
+    }
+   }
+  render() { 
+    return ( 
+      <Form>
+        <Row form>
+          <Col md={6}>
+            <FormGroup>
+              <Label for="firstName">First Name</Label>
+              <Input type="name" name="firstName" id="firstName" placeholder="First Name"/>
```

Comment:
make sure to also add the attribute `required` on each of these inputs so the user is prevented from submitting empty fields